docs/release/release-tasks.md

```diff
@@ -29,16 +29,16 @@ This comes down to changing occurrences of the old version to the new version, e
 1. Setup E2E tests for the new release:
    1. Goal is that our clusterctl upgrade tests are testing the right versions. For `v1.8` this means:
       - v1beta1: `v1.7` (will change with each new release)
-      - v1alpha4: `v0.8`
+      - v1beta1: `v1.7` (will change with each new release)
```

Comment:
Intended to be a duplicate of the above?

Reply:
Probably not :)